.github/workflows/dispatch-files-branches.yaml

```diff
@@ -13,6 +13,9 @@ on:
 env:
     REQUIRED_PHP_EXTENSIONS: redis
 
+permissions:
+  contents: read
```

Comment:
indent should be with 4 spaces. And there are more workflow files on the templates folder.

Reply:
For this PR I can fix these files.

Reply:
I have done that. PTAL. Thanks